resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -81,6 +80,13 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_EXECUTOR_SERVICE_ACCOUNT_NAME =
```

Comment:
Ur, this looks wrong.

Why do we backport an improvement JIRA, SPARK-30122 Allow setting serviceAccountName for executor pods, here?